### PR TITLE
following count of users

### PR DESCRIPTION
**Pull request by codesleep-Beperfect**

### CODE REVIEW

File: src/scrape_up/github/users.py [hunk 1]
```diff
@@ -25,6 +25,18 @@ def followers(self):
             message = f"{self.username} not found !"
             return message
 
+    def following(self):
+        """"
```
Comment (nikhil25803) [inline src/scrape_up/github/users.py:29]: Include this method in `documentation.md` file.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Comment by codesleep-Beperfect** [comment]
@nikhil25803 please look into it!
Seems alright to me

**Review by nikhil25803 — APPROVED** [review]
Looks good now.

File: src/scrape_up/github/users.py (hunk 1 above)
Comment (codesleep-Beperfect) [reply src/scrape_up/github/users.py:29]: @nikhil25803 added the function

**Pull request merged**